bee-test/Cargo.toml

```diff
@@ -12,7 +12,6 @@ homepage = "https://www.iota.org"
 
 [dependencies]
 bee-ternary = { version = "0.3.2-alpha", features = [ "serde1" ], path = "../bee-ternary" }
-
```

Comment:
Put it back please, for consistency we put a newline between our crates and external crates :)